### DISCUSSION TITLE
FP 1.11.1 prerelease 14: This application requires JRE 11 - 17

### DISCUSSION THREAD

**Discussion by dsmithcsepINCOSE56** [freeplane/freeplane#1006] [Help]
After reading about new FP 1.11.1-pre14 Release, I decided to install it because it appears to deal with some issues I was having with node positioning.

After download and install, I got an error that I needed JRE 11 - 17.

My FP 1.10.6 worked fine. - I had jre-8u333-windows-x64.exe with 5/15/2022 installation date.

The latest I was able to find today was jre-8u361-windows-x64.exe which I downloaded and installed.  It uninstalled the previous version.

I can't find any thing current on any JRE 11 - 17.

How to fix?

![2023-02-11 jjre-8u3xx-windows-x64](https://user-images.githubusercontent.com/123216787/218289949-f3900074-79f9-46a3-80ce-7de6821ced43.jpg)
![2023-02-11_JRE runtime 11 - 17 popup](https://user-images.githubusercontent.com/123216787/218289951-d808138c-eeaf-477a-99d1-3b0c46524698.jpg)

**Comment by abc16361** [comment]
You can try this one. I am using it. It's pretty good.
https://www.azul.com/downloads/?os=windows&architecture=x86-64-bit&package=jre

**Comment by dpolivaev** [comment]
You can use setup "with java" which contains embedded java and minimizes dependencies on external software

**Comment by dsmithcsepINCOSE56** [comment]
Dimitry,

Thanks so much!  That did the trick.

On my web browser, the "with Java" doesn't show on the list of 
downloads.  It didn't appear until I clicked the ""See All Activity"

See the attached.

While I had used Freemind off and on some before discovering Freeplane, 
my current project is taking me into the bowels of the application to 
fit my needs.

Thanks again for this tip!

Darold

++++++++++++++++++++++++++++++++++++

On 2023-02-12 15:13, Dimitry Polivaev wrote:

> You can use setup "with java" which contains embedded java and 
> minimizes dependencies on external software
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>

-- 
_Darold K. Smith: PE, CSEP_

Professional Engineer, Software and Electrical Engineering Texas

Certified Systems Engineering Professional, INCOSE

***@***.***

903 217-8259 (M)

Links:
------
[1] 
https://github.com/freeplane/freeplane/discussions/1006#discussioncomment-4951086
[2]
